pyproject.toml

```diff
@@ -57,7 +57,15 @@ addopts = [
   "--tb=native",
   "--strict-markers"
 ]
-
+filterwarnings = [
+  # elevate all deprecation warnings to errors
+  "error::PendingDeprecationWarning",
+  "error::DeprecationWarning",
+  "error::FutureWarning",
+  # temporary ignores
+  "ignore:conda.cli.install.print_activate:PendingDeprecationWarning:conda.env.env",
```

Comment:
We need this until https://github.com/conda/conda/pull/12771#discussion_r2071986105 is released